### PR TITLE
Develop DeepCTR models for Display Advertising Challenge on Kaggle

### PR DESCRIPTION
**Pull request by workingloong**
[Display Advertising Challenge](https://www.kaggle.com/c/criteo-display-ad-challenge) is a competition to prediction CTR.

### CODE REVIEW

File: .isort.cfg [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [settings]
 multi_line_output=3
 line_length=79
-known_third_party = PIL,docker,google,grpc,kubernetes,numpy,odps,pandas,pyspark,recordio,requests,setuptools,sklearn,tensorflow,yaml
+known_third_party = PIL,docker,elasticdl_antfin,google,grpc,kubernetes,numpy,odps,pandas,pyspark,recordio,requests,setuptools,sklearn,tensorflow,yaml
```
Comment (QiJune) [inline .isort.cfg]: Please remove this `elasticdl_antfin` deps

Comment (workingloong) [reply .isort.cfg]: Done!

File: model_zoo/dac_ctr/wide_dee_model.py [hunk 2]
```diff
@@ -0,0 +1,142 @@
+import tensorflow as tf
```
Comment (QiJune) [inline model_zoo/dac_ctr/wide_dee_model.py]: Please rename `wide_dee_model` to `wide_deep_model`.

File: model_zoo/dac_ctr/feature_config.py [hunk 3]
```diff
@@ -0,0 +1,141 @@
+STANDARDIZED_FEATURES = [
```
Comment (QiJune) [inline model_zoo/dac_ctr/feature_config.py:1]: We have a model `cenus_wide_deep` in the model zoo already. So how about naming the directory from `dac_ctr` to `criteo_wide_deep`? We may add `criteo_xdeepfm` in the later.

File: elasticdl_preprocessing/layers/normalizer.py [hunk 4]
```diff
@@ -59,6 +59,7 @@ def call(self, inputs):
         return normalized_tensor
 
     def _normalize_fn(self, x):
-        subtractor = tf.cast(self.subtractor, x.dtype)
-        divisor = tf.cast(self.divisor, x.dtype)
+        x = tf.cast(x, tf.float64)
+        subtractor = tf.cast(self.subtractor, tf.float64)
+        divisor = tf.cast(self.divisor, tf.float64)
```
Comment (skydoorkai) [inline elasticdl_preprocessing/layers/normalizer.py]: Deep learning usually does not use float64. What is the reason to use it instead of float32?

**Review by skydoorkai — COMMENTED** [review]
Any doc to explain how to prepare the training data? Use the raw data or do we need to convert the data to RecordIO?

File: elasticdl_preprocessing/layers/normalizer.py (hunk 4 above)
Comment (workingloong) [reply elasticdl_preprocessing/layers/normalizer.py]: I have fixed it in the #1952

File: model_zoo/dac_ctr/feature_config.py (hunk 3 above)
Comment (workingloong) [reply model_zoo/dac_ctr/feature_config.py:1]: The `census_wide_deep` is a demo to design feature transformation. We are considering to develop a tool for users to transform feature in #1950. We can place the `wide_deep` and `xdeepfm` model to a folder and use them in the train definition file for census and dac. We can first merge the PR and then refactor them.

**Comment by workingloong** [comment]
> Any doc to explain how to prepare the training data? Use the raw data or do we need to convert the data to RecordIO?

Yeah， I have added README.md

**Review by QiJune — APPROVED** [review]
Excellent！

**Pull request merged**